algorithms/ar-msrt/javascript/merge_sort.js

```diff
@@ -0,0 +1,36 @@
+function merge(array1, array2) {
```

Comment:
Codacy found an issue: [Function 'merge' has a complexity of 8. Maximum allowed is 5.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6336020)

Reply:
ignore this issue.